services/QuillLMS/spec/queries/progress_reports/district_standards_reports_spec.rb

```diff
@@ -3,308 +3,194 @@
-          end
+  # standards_report_query(3945512, 6)
+  let(:example_json2) do
+    {
+      "kind"=>"bigquery#queryResponse",
+      "schema"=>{
+        "fields"=>[
+          {"name"=>"id", "type"=>"INTEGER", "mode"=>"NULLABLE"},
+          {"name"=>"name", "type"=>"STRING", "mode"=>"NULLABLE"},
+          {"name"=>"standard_level_name", "type"=>"STRING", "mode"=>"NULLABLE"},
+          {"name"=>"total_activity_count", "type"=>"INTEGER", "mode"=>"NULLABLE"},
+          {"name"=>"total_student_count", "type"=>"INTEGER", "mode"=>"NULLABLE"},
+          {"name"=>"proficient_count", "type"=>"INTEGER", "mode"=>"NULLABLE"},
+          {"name"=>"timespent", "type"=>"FLOAT", "mode"=>"NULLABLE"}
+        ]
+      },
+      "jobReference"=>{
+        "projectId"=>"analytics-data-stores", "jobId"=>"job_hdxcHeTAbIgInJ5nZvPUwryG853s", "location"=>"us-central1"
+      },
+        "totalRows"=>"1",
+        "rows"=>[
+          {
+            "f"=>[
+              {"v"=>"6"},
+              {"v"=>"1.1i Frequently occurring prepositions"},
+              {"v"=>"CCSS: Grade 1"}, {"v"=>"4"}, {"v"=>"103"},
+              {"v"=>"103"},
+              {"v"=>"51010.0"}
+            ]
+          }
+        ],
+        "totalBytesProcessed"=>"7823614651",
+        "jobComplete"=>true,
+        "cacheHit"=>false
+    }
+  end
 
-          context 'percentage1 meets threshold' do
-            let(:percentage1) { proficient_threshold }
 
-            it { expects_results }
-            it { expect(subject[0]["proficient_count"]).to eq 1 }
-          end
+  context '#standards_report_query' do
+    let(:assigned_student_ids) { [] }
 
-          context 'student1 has another finished another activity_session with activity1' do
-            before do
-              create(
-                :activity_session,
-                :finished,
-                activity: activity1,
-                percentage: percentage2,
-                timespent: timespent2,
-                classroom_unit: classroom_unit,
-                user: student1
-              )
-            end
+    context 'integration', :external_api do
+      around do |a_spec|
+        VCR.configure { |c| c.allow_http_connections_when_no_cassette = true }
+        a_spec.run
+        VCR.configure { |c| c.allow_http_connections_when_no_cassette = false }
+      end
 
-            it { expects_results }
-            it { expect(subject[0]["proficient_count"]).to eq 1 }
-          end
-        end
+      it 'tests end to end' do
+        expected = [
+          {
+          "id"=>7,
+          "name"=>"1.1b Use common, proper, and possessive nouns",
+          "standard_level_name"=>"CCSS: Grade 1",
+          "total_activity_count"=>1,
+          "total_student_count"=>82,
+          "proficient_count"=>82,
+          "timespent"=>nil
+          }
+        ]
+
+        report = described_class.new(admin.id)
+        query_result = report.standards_report_query(3945512, 7)
```

Comment:
The magic numbers: `3945512`  and `7` should be extracted as constants, especially `3945512` since it's referenced a few times more below.